docs/docs/installation/docker/index.md

```diff
@@ -1,21 +1,54 @@
-# Install Mathesar Webserver via Docker
+# Install Mathesar web server via Docker
+
+Use our [official Docker image](https://hub.docker.com/r/mathesar/mathesar-prod/tags): `mathesar/mathesar-prod:latest` hosted on Docker Hub to run Mathesar.
+
+!!! warning "Limitations"
+    This installation procedure is intended for users who want to run a bare-bones version of the Mathesar web server.
+
+    It is assumed you already have a database server and services like a reverse proxy typically needed for running a production setup. If you don't have those, please use the [Docker Compose installation documentation](../docker-compose/index.md).
 
 
 ## Prerequisites
-!!! warning ""
-    This installation procedure is intended for users who want to run a bare-bones version of the Mathesar web server. It is assumed you already have services like a reverse proxy needed for running a production server. To set up a proper production server, please refer to the [Docker Compose Installation Documentation](../docker-compose/index.md).
-
-- You need [Docker](https://docs.docker.com/get-docker/)
-    - We have tested with Docker v23. Older versions may not work.
-- Permission to run Docker containers
-- A Postgres database. Ensure the external database can accept network connections from your Mathesar server.
-- Have the following information handy before installation:
+
+### Operating System
+You can install Mathesar using this method on Linux, MacOS, and Windows.
+
+### Access
+You should have permission to run Docker containers on the system.
+
+### Software
+You'll need to install **[Docker](https://docs.docker.com/desktop/)** v23+
+
+### Databases
+
+#### Database for Mathesar's internal usage
+You'll need to:
+
+- create a PostgreSQL database for Mathesar's internal usage.
+- create a database user for Mathesar to use. The user should be a `SUPERUSER`, [see PostgreSQL docs for more information](https://www.postgresql.org/docs/13/sql-createrole.html).
+- ensure that this database can accept network connections from the machine you're installing Mathesar on.
+- Have the following information for this database handy before installation:
     - Database hostname
     - Database port
     - Database name
-    - Database username _(should exist and be a `SUPERUSER` [more info](https://www.postgresql.org/docs/13/sql-createrole.html))_
+    - Database username
     - Database password
 
+!!! warning "Create"
+    This installation procedure is intended for users who want to run a bare-bones version of the Mathesar web server. 
```

Comment:
I find this new admonition box a little confusing. The title "Create" doesn't seem to be warning me of anything on its own, so it's confusing while I skim. Then, then content in the box is a duplicate of the content at the top of the page. Can we either remove the box or add some clarity?

Reply:
This appears to be some sort of typo, I was running out of steam by the time I got to this page. I'll fix.

Reply:
I removed this in https://github.com/centerofci/mathesar/pull/2907/commits/4e7ddb6b6bb06399b350fef1613b9497b754ab42. I think I was moving text around and accidentally copied instead of cut or something like that.

Reply:
Nice catch btw